pipeline/unsigned-container-image-listener.groovy

```diff
@@ -90,7 +90,9 @@ node(nodeName) {
                 "email": "ceph-qe@redhat.com"
             ],
             "build": [
-                "repository": compose.repository
+                "repository": compose.repository,
+                "composes": releaseDetails.latest.composes,
+                "version": cephVersion
```

Comment:
Would it make sense to put this information under `redhat`? that is
```
"build": [
  "redhat": [
    "repository": compose.repository,
    "composes": releaseDetails.latest.composes,
    "version": cephVersion
  ]
]
```

Reply:
As discussed, not needed.